Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN curl -sSL https://install.python-poetry.org | python3 -
 ENV PATH=$PATH:/root/.local/bin/
 WORKDIR /app
 COPY pyproject.toml poetry.toml /app/
-RUN poetry install
+RUN /root/.local/bin/poetry install
```

Comment:
Once we worked out the curl issue, it's worth noting that we can switch this back to just `poetry install`, but equally no real harm in having the full path (just a touch of repetition)